internal/app/encryption.go

```diff
@@ -88,16 +88,22 @@ func NewBcrypt(key []byte) string {
 }
 
 // CreateHash ...
-func CreateHash(key string) string {
+func CreateHash(key string) (string, error) {
 	hasher := md5.New()
-	hasher.Write([]byte(key))
-	return hex.EncodeToString(hasher.Sum(nil))
+	if _, err := hasher.Write([]byte(key)); err != nil {
+		return "", err
+	}
+	return hex.EncodeToString(hasher.Sum(nil)), nil
 }
 
 // Encrypt ..
 func Encrypt(dataStr string, passphrase string) []byte {
 	dataByte := []byte(dataStr)
-	block, _ := aes.NewCipher([]byte(CreateHash(passphrase)))
+	hash, err := CreateHash(passphrase)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Please do not use panic anywhere in the application.

Reply:
I added panic just because there is already a usage of a panic a couple of lines below. Should I delete those panics as well?

Reply:
Yes, please.

Reply:
Deleted pacic statements